{{cookiecutter.repo_name}}/devtools/conda-envs/test_env.yaml

```diff
@@ -8,7 +8,8 @@ dependencies:
     # Testing
   - pytest
   - pytest-cov
+  - codecov
 
     # Pip-only installs
-  - pip:
-    - codecov
+  # - pip:
```

Comment:
This is the part that will need `if conda` around them like [here](https://github.com/MolSSI/cookiecutter-cms/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/devtools/travis-ci/before_install.sh#L33).